docker_files/install_pyflame.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
Prolly just go ahead and delete this.  We do not use it anymore under any circumstances.

Reply:
Oh good to know. It's the only reason we need to expose the Python include directories, etc. when installing Python.

Reply:
great news!